grails5 upgrade

niravassar opened the pull request.

Commit:
grails5 upgrade

sdelamo commented on complete/build.gradle:
this should be decoupled from grails version

sdelamo commented on complete/build.gradle:
this should be 2.1.1

sdelamo commented on complete/gradle.properties:
grails version should be 5.0.1

sdelamo commented on complete/gradle.properties:
this should be GORM 7.1.0

sdelamo commented on initial/build.gradle:
this should be decoupled from grails version

sdelamo commented on initial/build.gradle:
this should be 2.1.1

sdelamo commented on initial/gradle.properties:
grails version should be 5.0.1

sdelamo commented on initial/gradle.properties:
this should be GORM 7.1.0

jamesfredley committed:
Update README to point at canonical guide on grails.apache.org (refs apache/grails-static-website#354)

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
